### DISCUSSION TITLE
Build QT6 issue on macOS 11.5 Big Sur

### DISCUSSION THREAD

**Discussion by sam830917** [Homebrew/discussions#1977] [Getting started]
I am working on porting my project from qt5 to qt6.
When I build my project, it will show up the following error

`Sorry, "MyProject" cannot be run on this version of macOS. Qt requires macOS 11.0.0 or later, you have macOS 10.16.0.`
I installed the Qt6 by homebrew and build it by XCode 12.
MacOS version is 11.5.

For testing, I installed the Qt6 by Qt online installer.
I created the new project on Qt Creator which install by homebrew, it shows the same error.
But I installed Qt by Qt online installer, and run a random example project, it works properly.
After that, I have noticed that when using desktop kit, it will show the same error, but using Qt 6.1.2 kit it works fine which was installed by Qt online installer.
I am wondering if it is homebrew issue or what.
I am looking for the solution for a while, but still cannot solve the issue.

**Comment by SMillerDev** [comment]
Looks like you have `SYSTEM_VERSION_COMPAT` set somewhere and macOS is reporting being macOS 10.16.
